gnome-initial-setup/pages/password/gis-password-page.c

```diff
@@ -194,6 +203,14 @@ on_focusout (GisPasswordPage *page)
 }
 
 static void
+reminder_changed (GtkWidget      *w,
+                  GParamSpec     *pspec,
+                  GisPasswordPage *page)
+{
+    gis_page_set_complete (GIS_PAGE (page), page_validate (page));
```

Comment:
Watch the indentation here too, and the alignment of function parameters.

(I know it was probably broken before and you just cherry-picked it, but now seems like a good time to fix it.)